api/src/test/java/io/kaoto/backend/api/resource/v2/IntegrationsResourceTest.java

```diff
@@ -369,10 +369,11 @@ void beans() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"Camel Route#route-multi.yaml", "KameletBinding#kamelet-binding-multi.yaml",
```

Comment:
Why are we removing the kamelet bindings here?

Reply:
ugh was by mistake, got it back.